components/icons/LightNeed.js

```diff
@@ -0,0 +1,34 @@
+import PartialShade from "@/public/half-shade.svg";
+import Shade from "@/public/sun-shade.svg";
+import Sun from "@/public/sun.svg";
+import styled from "styled-components";
+
+export default function LightNeed({ lightNeed }) {
+  return (
+    <LightNeedWrapper>
+      <StyledShade $isShade $isLight={lightNeed === "Full Shade"} />
```

Comment:
```suggestion
      <StyledShade $isLight={lightNeed === "Full Shade"} />
```
I think you don't need that prop.